### PR TITLE
Add Akaunting

### PR DESCRIPTION
**Pull request by denisdulici**
Accounting software https://akaunting.com

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -10774,15 +10774,33 @@
...
+			"script": "app\\.raychat\\.io/scripts/js",
+			"website": "https://raychat.io"
+		},
+		"Akaunting": {
+			"cats": [
+				"54"
+			],
+			"html": [
+				"<link[^>]akaunting-green\\.css",
+				"Powered By Akaunting"
```
Comment (AliasIO) [inline src/apps.json]: Please update this pattern to include a HTML tag. HTML patterns should not match plain text.

**Comment by denisdulici** [comment]
Added Elbert.

**Comment by denisdulici** [comment]
@AliasIO any update about our PR?

File: src/apps.json [hunk 2]
```diff
@@ -10789,24 +10789,42 @@
...
+			"html": [
+				"<link[^>]akaunting-green\\.css",
+				"Powered By Akaunting: <a [^>]*href=\"https?://(?:www\\.)?akaunting\\.com[^>]+>"
+			],
+			"headers": {
+				"X-Akaunting": "Free Accounting Software"
+			},
+			"implies": [
+				"PHP",
+				"Laravel"
```
Comment (jvoisin) [inline src/apps.json]: `Laravel` already implies `PHP`.

File: src/apps.json [hunk 3]
```diff
@@ -10789,24 +10789,42 @@
...
+		"Akaunting": {
+			"cats": [
+				"54"
+			],
+			"html": [
+				"<link[^>]akaunting-green\\.css",
+				"Powered By Akaunting: <a [^>]*href=\"https?://(?:www\\.)?akaunting\\.com[^>]+>"
+			],
+			"headers": {
+				"X-Akaunting": "Free Accounting Software"
```
Comment (jvoisin) [inline src/apps.json]: Please use `"X-Akaunting": "^Free Accounting Software$"` if possible.

Comment (jvoisin) [inline src/apps.json]: Isn't akaunting a bit [too new](https://akaunting.com/blog/introducing-akaunting) (it's only 3 months old) to have a significant number of users?

File: src/apps.json [hunk 4]
```diff
@@ -10789,24 +10789,42 @@
...
-    "Mautic": {
-      "cats": [
-        "32"
-      ],
-      "env": "^MauticTrackingObject$",
-      "icon": "mautic.svg",
-      "script": "[^a-z]mtc.*\\.js",
-      "website": "https://www.mautic.org/"
-    }
+		"Raychat": {
```
Comment (jvoisin) [inline src/apps.json]: This is unrelated to your pull-request :/

Comment (denisdulici) [reply src/apps.json]: Tried to help further, it's space was wrong..

File: src/apps.json (hunk 3 above)
Comment (denisdulici) [reply src/apps.json]: Any reason? It will never come anything else before and after.

File: src/apps.json (hunk 4 above)
Comment (jvoisin) [reply src/apps.json]: Please open a pull-request per topic to ease the review ♥
(And to speed up their merging)

File: src/apps.json (hunk 3 above)
Comment (denisdulici) [reply src/apps.json]: Not sure what have this to do with this PR?

File: src/apps.json (hunk 2 above)
Comment (denisdulici) [reply src/apps.json]: Ok will remove PHP.

**Comment by denisdulici** [comment]
Can you plz check it now?

File: src/apps.json (hunk 3 above)
Comment (AliasIO) [reply src/apps.json]: `<link[^>]` Should be `<link[^>]+` or it won't work.

@denisdulici, @jvoisin is referring to this:

https://wappalyzer.com/docs#new-application
> Only widely used applications should be added. When creating a pull request, please include ten or more links to websites that use the application, a GitHub page with at least 1,000 stars or anything that will help establish the size of the user base.

Comment (denisdulici) [reply src/apps.json]: Elbert, can see apps with no links and only 18 GH stars been accepted!

File: src/apps.json [hunk 5]
```diff
@@ -10900,7 +10900,22 @@
...
+		"Akaunting": {
+			"cats": [
+				"55"
+			],
+			"html": [
+				"<link[^>]+akaunting-green\\.css",
+				"Powered By Akaunting: <a [^>]*href=\"https?://(?:www\\.)?akaunting\\.com[^>]+>"
+			],
+			"headers": {
+				"X-Akaunting": "Free Accounting Software"
```
Comment (jvoisin) [inline src/apps.json]: Would it be possible to use `"X-Akaunting": "^Free Accounting Software$"` instead?

**Comment by AliasIO** [comment]
Thanks!
